### PR TITLE
chore(showcase): Add EF

### PR DESCRIPTION
**Pull request by jhpmatos**
## Description

Add EF design portfolio website to showcase

### CODE REVIEW

File: docs/sites.yml [hunk 1]
```diff
@@ -9059,3 +9059,13 @@
   built_by: Paul Scanlon
   built_by_url: http://www.pauliescanlon.io
   featured: false
+- title: EF Design
+    main_url: "https://ef.design"
```
Comment (muescha) [inline docs/sites.yml]:
```suggestion
    main_url: https://ef.design
```

File: docs/sites.yml [hunk 2]
```diff
@@ -9059,3 +9059,13 @@
...
+    url: "https://ef.design"
```
Comment (muescha) [inline docs/sites.yml]:
```suggestion
    url: https://ef.design
```

**Comment by laurieontech** [comment]
Hi @muescha, thanks for the help. I did want to note that this PR had someone assigned to it and we'd like that to be respected. So if you can keep an eye out for that going forward we'd appreciate it. Thanks!

**Comment by muescha** [comment]
ah ok - i will look for assignees
_(i only looked out for the label `topic: internal` (https://github.com/gatsbyjs/gatsby/pull/19965#issuecomment-564117391))_

**Pull request merged**